src/Styleguide/Components/Tabs.tsx

```diff
@@ -185,3 +185,15 @@ const TabContainer = styled.div`
 const ActiveTabContainer = styled.div`
   ${styles.activeTabContainer};
 `
+
+const SupWrapper = styled.sup`
+  margin-left: 2px;
+`
+
+export const Sup: React.SFC<{}> = ({ children }) => (
```

Comment:
Is `Tabs` the best place for this component? It seems it might be something that should stand on its own.

Reply:
Good point. It was originally named `TabSuperscript` and I renamed it to `Sup` but its implementation is very specific to this context so it shouldn't be its own thing. I can rename it back to something like `TabSup` if it is confusing, and/or differ the decision about if we want e general purpose `Sup` component  for now.